packages/blade/src/components/Checkbox/Wrapper.native.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```

Comment:
this is a general comment, instead of creating too many small small platform dependent components can't we create just one platform dependent component? because we just need an entry point. right now there's a lot of abstraction and as a dev it take a lot of cognitive load to keep jumping files to even figure out a small thing

Reply:
*cough cough Box cough*